src/TokenHandler.mo

```diff
@@ -349,6 +351,12 @@ module TokenHandler {
     /// retrieve the sum of all successful consolidations
     public func totalConsolidated() : Nat = totalConsolidated_;
 
+    /// retrieve the sum of all deductions from main account of the token handler
+    public func totalWithdrawnFromMain() : Nat = totalWithdrawnFromMain_;
+
+    /// retrieve the calculated balance of main account of the token handler
+    public func handlerMainAccountFunds() : Nat = totalConsolidated_ - totalWithdrawnFromMain_;
```

Comment:
Let's shorten it to `consolidatedFunds()`

Reply:
Done